dll/win32/comdlg32/lang/cdlg_Bg.rc

```diff
@@ -369,7 +369,7 @@ FONT 8, "MS Shell Dlg"
 
     GROUPBOX        "Копия",         grp2, 160, 92, 120,64, WS_GROUP
     LTEXT           "Брой &копия:",stc5,168,105,68,8
-    ICON            "",               ico3, 170,131,  76,24, WS_GROUP | SS_CENTERIMAGE
+    ICON            "",               ico3, 210,131,  76,24, WS_GROUP// | SS_CENTERIMAGE
```

Comment:
If `SS_CENTERIMAGE` is wrong, please remove it. If it's correct but there's a bug in ROS that means we can't use it, that deserves a comment.

Reply:
We definitely can't use SS_CENTERIMAGE. I made a test program for windows and it has exactly the same behavior; when using SS_CENTERIMAGE the icon is not shown.